### ISSUE TITLE
ZoneOffset not fully qualified

### ISSUE THREAD

**Issue by joke** [mapstruct/mapstruct#1569]
Hi,

in 1.3.0.Beta1 the conversion between Date and LocalDate is currently working because ZoneOffset is not fully qualified.

```
if ( as.getPromotionDate() != null ) {
    article.setPromotionDate( java.time.LocalDateTime.ofInstant( as.getPromotionDate().toInstant(), ZoneOffset.UTC ).toLocalDate() );
}
```

```
cannot find symbol
  symbol:   variable ZoneOffset
```

**Comment by filiphr** [comment]
And when we thought that something like this won't happen again 😄. Seems like we missed a place. Thanks for reporting this. We are going to fix it with the next release.

One question, do you have `org.joda.time.LocalDateTime` in your class (asking because of the FQN from the `java.time.LocalDateTime`?.

You can workaround this bug by using `@Mapper(imports = ZoneOffset.class)`

**Comment by filiphr** [comment]
This has been resolved in PR #1575. It will be in the next release.

@joke in case you want to follow the issue with the FQN have a look at #1576. We pride in having a nice readable code and having that FQN was a sore to my eyes 😉
